buildbot-conf/cordova-config.json.sample

```diff
@@ -0,0 +1,43 @@
+{
+    "couchdb": {
+        "uri":  "http://localhost:5984"
+    },
+    "dashboard": {
+        "port": 8088
+    },
+    "app": {
+        "entry":   "autotest/pages/all.html",
+        "release": "master",
+        "timeout": 600
```

Comment:
No, because the new `cordova.conf` doesn't use them that way anymore. This config runs all the builders it has.